pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2567,4 +2587,72 @@
       ``[secure port host]
     ==
   ==
+++  return-static-data
```

Comment:
Probably rename as `+build-http-event` and make `+return-static-data-on-duct` use it.